components/cam/tools/mkatmsrffile/README

```diff
@@ -1,25 +1,21 @@
-Atmospheric drydeposition at the surface depends on certain surface
-properties including soil and land use properties.  In most cases
-these calculations can be handled in the land model and passed to he
-atmosphere through the coupler.  This is the default namelist setting
-drydep_method='xactive_lnd'.  However with modal areosols this method
-is not adequate and we must recalculate these fields in the atmosphere
-(see subroutine interp_map in mo_drydep.F90).  For unstructured grids
-it was determined to create this offline interpolation tool rather
-than generalize the subroutine interp_map.  
-
-1. Locate or create scrip mapping files for the 1x1 degree input grids and
-   the desired model output grid.
+Atmospheric drydeposition at the surface depends on certain surface properties including soil and land use properties.  In most cases, these calculations can be handled in the land model and passed to the atmosphere through the coupler.  This method is the default namelist setting drydep_method='xactive_lnd'. 
```

Comment:
I don't particularly like the change to make the text all one line here. Is there a reason for this? It makes it hard to read unless you have your editor configured a certain way.